prisma/schema.prisma

```diff
@@ -144,6 +146,11 @@ model Chapter {
   // Google Drive API related fields
   chapterFolder String   @default("")
   permissions   String[]
+
+  chapterRequestId String @db.ObjectId @unique
+  chapterRequest ChapterRequest @relation(fields: [chapterRequestId], references: [id], onDelete: Cascade)
```

Comment:
Shouldn't the onDelete cascade be on the ChapterRequest field here? Wouldn't  the onDelete here just delete the chapter is the chapterrequest is deleted?

I assume what you are trying to do here is delete the ChapterRequest if the chapter is deleted so I believe the onDelete should be on the chapterrequest side.